docs/1-2-design.md

```diff
@@ -11,7 +11,7 @@ One of the major highlights of React Native is its simplicity around creating ap
 
 However, rather than "write once, run anywhere", the philosophy of React Native has always been "learn once, write anywhere". This subtle difference means that React Native apps should be tailored for each platform, rather than being identical everywhere.
 
-From a UI perspective, as platforms have somewhat different visual styles, UI paradigms, or even technical capabilities, it makes sense to start with a common UI foundation, and then tweak it for each platform.
+From a UI perspective - as platforms have somewhat different visual styles, UI paradigms, or even technical capabilities - it makes sense to start with a common UI foundation, and then tweak it for each platform.
```

Comment:
How about....

"As platforms have somewhat different visual styles, UI paradigms, or even technical capabilities, it makes sense, from a UI perspective, to start with a common UI foundation, and then tweak it for each platform."

Reply:
@JoelMarcey the issue I have with the suggestions is that the commas are used in multiple fashions, which is slightly confusing. 

> "...somewhat different visual styles, UI paradigms, or even technical capabilities..."

Uses commas to separate a list of phrases 

> "...it makes sense, from a UI perspective, to start with a common UI foundation..."

Uses commas to set off a clause not essential to the meaning of the sentence.

I think the use of commas in different ways in one sentence can be confusing.

Reply:
@stepharr I think my suggested restructure reads ok, but I definitely get your point. How about you break it up into two sentences then? Like...

"Platforms can have somewhat different visual styles, UI paradigms, or even technical capabilities. Thus, it makes sense to start with a common UI foundation, and then tweak it for each platform."